azure-function/dotnet/Models/VehicleId.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GetBusData.Models;
+
+public sealed class VehicleId
+{
+    [JsonProperty("id")]
```

Comment:
I don't think this `JsonProperty(...)` is necessary because the function app applies camelCasing by default.